api/internal/core/store/selector.go

```diff
@@ -0,0 +1,144 @@
+/*
```

Comment:
???

Reply:
where is this file come from? please give the github repo

Reply:
some code from:
https://github.com/kubernetes/dashboard/tree/master/src/app/backend/resource/dataselect
not whole file.